docs/learn/software_overview.md

```diff
@@ -1,53 +1,54 @@
+:   For the most part, you are either going to use your CNC machine to create a part (e.g. a wooden or metal gear) or for something         artistic (e.g. a sign or carving). If the circumstance is that you are needing to create the design from scratch you will likely be       using CAD or Vector Graphics Editor software to do so.  It is worth noting that this is NOT a certainty.  Many people may already       have these designs in hand provided from some other source.  Regardless, the output of this step (and input to the next) is             typically either a 2D file (like a .DXF or .SVG) or sometimes (but not often) a 3D file (like a .STL).
 
-[CAM: Computer Assisted Machining](#cam-computer-assisted-machining)
-:   This is the step where you take your design, and describe which lines to trace, which areas to
-    cut out, and which parts to leave. CAM also is where you will define what kinds of bits you
-    have, and how fast and deep you want to move them through the material. The output of this step
-    is g-code (in a .gcode or .nc file).
+[Toolpathing/CAM](#toolpathing-cam)
+:   This is the step where you take your design, and describe which lines to engrave, which areas to
+    cut out, and which parts to leave. It defines what "paths" your "tool" is going to take.  CAM also is where you will 
+    define what kinds of CNC bits you have, and how fast and deep you want to move them through the material. The output of
+    this step is g-code (in a .gcode or .nc file). G-code is the "language" that your CNC understands to do perform its movements.
 
 [G-code Sender](#g-code-sender)
 :   The g-code is just a list of instructions that define where the tool should go. The software
-    used to "play" these instructions is the g-code sender. There are many options for this step, so
-    do what feels comfortable. The g-code sender needs to be well matched with your controller.
+    used to "play" these instructions is the g-code sender and communicates with the controller. There are a number of options 
+    for this software.  Most people have a personal preference along with it being well matched with your controller.
 
-[Controller](#controller)
-:   A.k.a. "Firmware"
-    The controller is responsible for taking g-code instructions and managing the precise movements
-    of the motors. There are other jobs, but they are all related to this one function. In our CNC
-    machines, the most common is Marlin, and the second is Grbl.
+[Milling/Controller/Firmware](#milling-controller-firmware)
+:   The controller is responsible for taking g-code instructions and managing the precise movements
+    of the motors. There are other jobs, but they are all related to this one function. There is firmware (software on a chip) that facilitates this.  In the CNC world there are many "flavors" of this firmware.  In our CNC machines, the most common flavors are Marlin and Grbl.
 
-## CAD: Computer Assisted Design
+## Design-CAD-VGE
```

Comment:
You're right on that inkscape and illustrator are not CAD. Maybe this section title and above will be clearer if we just call it "Design" and then talk about CAD/VGE in the details?

Reply:
I am flexible but agree.  You have picked up on the **exact** nature of the change that I was think.  I personally came into this only wishing to make signs.  As such I have only needed to use something to convert or design my sign, but as it turns out, I have never need CAD specifically.

Reply:
```suggestion
## Design
```